### PR TITLE
Unified Xamarin

### LINKED ISSUE #4: Bouncy Castle PCL & Xamarin Unified Api
Hello,

Thanks for your PCL. We have been using it successfully with Xamarin.Android & Xamarin.iOS projects.

However, recently after converting our Xamarin.iOS project to support Unified API, the project doesn't compile due to crypto assembly. This issue has cropped up only after converting the project to Unified API. Earlier Bouncy Castle was working with Xamarin.iOS.

Is this a known issue? Is there a fix/workaround already?

Regards,
Gautam Jain

### PR DESCRIPTION
**Pull request by bscheiman**
Added Xamarin-specific bits for .nuspec.

Fixes #4

### CODE REVIEW

File: Portable.BouncyCastle.nuspec [hunk 1]
```diff
@@ -35,9 +35,9 @@
         <tags>bouncycastle, cryptography, encryption, security, PCL, wp8, wpa81, net4, sl5, monotouch, monoandroid, xamarin, xamarin.ios</tags>
     </metadata>
     <files>
-        <file src="Crypto\bin\Release\crypto.xml" target="lib\portable-net4+sl5+wp8+win8+wpa81\crypto.xml" />
-        <file src="Crypto\bin\Release\crypto.pdb" target="lib\portable-net4+sl5+wp8+win8+wpa81\crypto.pdb" />
-        <file src="Crypto\bin\Release\crypto.dll" target="lib\portable-net4+sl5+wp8+win8+wpa81\crypto.dll" />
+        <file src="Crypto\bin\Release\crypto.xml" target="lib\portable-net4+sl5+wp8+win8+wpa81+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10\crypto.xml" />
```
Comment (onovotny) [inline Portable.BouncyCastle.nuspec]: Oh the joys of platform multiplication...and MAX_PATH limits :)

Comment (bscheiman) [inline Portable.BouncyCastle.nuspec]: Hah... yeah, I know. I can make it a separate folder if you prefer that.

**Comment by onovotny** [comment]
I just pushed this to NuGet

**Comment by bscheiman** [comment]
Thanks!

**Pull request merged**